src/js/utils/MesosStateUtil.js

```diff
@@ -2,8 +2,34 @@ import Util from './Util';
 
 const RESOURCE_KEYS = ['cpus', 'disk', 'mem'];
 
+function setSchedulerName(name, tasks) {
+  return tasks.map(function (task) {
+    let newTask = Object.assign({}, task);
+
+    newTask.scheduler = name;
+
+    return newTask;
+  });
+}
+
 const MesosStateUtil = {
 
+  assignSchedulerNameToTasks(state) {
```

Comment:
@mlunoe I agree that we should use the schedule ID to accurately identify tasks lunched by a framework but I don't see any issue with adding the framework name to the tasks. 
I would use the framework name for now and switch to the scheduler ID once we need to support multiple Marathon instances as it required extra API calls to fetch the respective IDs. Are you ok with that?